### DISCUSSION TITLE
--hostname not listed on Switches that work with --burn

### DISCUSSION THREAD

**Discussion by dshaw619** [gitbls/sdm#42] [Ideas]
Just a minor documentation note:  On the list of "Switches that work with --burn" (https://github.com/gitbls/sdm/wiki/Burn-Scripts#switches-that-work-with---burn) --hostname isn't listed although is does work.

**Comment by gitbls** [comment]
Good catch! It does now...

**Reply by dshaw619** [reply]
Thanks.
